src/api/player/client/show3DText.lua

```diff
@@ -8,21 +8,26 @@
 ---@return void
 ---@public
 function API_Player:show3DText(x, y, z, text, font, scale)
-    local onScreen <const>, x <const>, y <const> = World3dToScreen2d(x, y, z)
+    local onScreen <const>, x <const>, y <const> = GetScreenCoordFromWorldCoord(x, y, z)
     if not (scale) then
         scale = 1
     end
     if not (font) then
         font = 4
     end
-    if (onScreen) then
-        SetTextScale(0.0 * scale, 0.55 * scale)
-        SetTextFont(font)
-        SetTextProportional(1)
-        SetTextColour(255, 255, 255, 215)
-        SetTextEntry("STRING")
-        SetTextCentre(true)
-        AddTextComponentString(text)
-        DrawText(x, y)
+    if (not onScreen) then
+        return
     end
+
+    SetTextScale(0.0 * scale, 0.55 * scale)
```

Comment:
What is the purpose of multiplication with 0?

Reply:
I don't know I didn't write, I think it's because of the float type.